fabfile.py

```diff
@@ -130,6 +130,15 @@ def setup():
                 run("ln -s -T default_images images")
                 run("ln -s -T default_styles styles")
 
+    # Create a file for personal Fabric commands
+    # but only if it's not already there
+    if not files.exists(env.rbase + "fabfile_personal.py"):
+        files.upload_template(
+            join(env.lbase, "deploy", "config_templates", "fabfile_personal.py"),
+            env.rbase + "fabfile_personal.py",
+            backup=False
+        )
+
```

Comment:
I'd rather not create a personal `fabfile` by default. I think it would be cleaner for people who don't want to use one.